geoips/plugins/modules/output_checkers/netcdf.py

```diff
@@ -182,6 +193,15 @@ def outputs_match(plugin, output_product, compare_product):
     try:
         xarray.testing.assert_allclose(compare_xobj, out_xobj)
     except AssertionError as resp:
+        message = "BAD GeoIPS NetCDF files do not match within tolerance"
+        log_with_emphasis(
+            LOG.interactive,
+            [
+                message,
+                f"output_product: {output_product}",
+                f"compare_product: {compare_product}",
+            ],
+        )
         LOG.interactive("    ****************************************************")
         LOG.interactive(
             "    *** BAD GeoIPS NetCDF files do not match within tolerance *****"
```

Comment:
Is this going to print the same "Bad GeoIPS NetCDF" message twice?

Reply:
I think it is - it looks like the original "LOG.interactive" messages were not removed when the log_with_emphasis call was added?